stages/org.osbuild.zip

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python3
+"""
+Assembles the tree into a zip archive named `filename`
```

Comment:
According to https://matrix.to/#/!yAOrVhosqfnOcdXiYb:fedoraproject.org/$Vp5m8A5THRcf9CdQxnhuF0GpyM1KjAalMlGc200eIeg?via=fedoraproject.org&via=matrix.org&via=fedora.im we'll also need a line here that says something like:

```
Buildhost commands used: `zip`
```